components/Layout/Header.js

```diff
@@ -127,11 +128,13 @@ class Header extends React.Component {
                     </b.MenuItem>
                   </LinkContainer>
                 )}
-                <LinkContainer to="/forum">
-                  <b.MenuItem>
-                    <font className={s.navLinks}>Forum</font>
-                  </b.MenuItem>
-                </LinkContainer>
+                {showForum && (
+                  <LinkContainer to="/forum">
+                    <b.MenuItem>
+                      <font className={s.navLinks}>Forum</font>
+                    </b.MenuItem>
+                  </LinkContainer>
+                )}
```

Comment:
`<font>` tag is outdated and doesn't do anything in HTML5 anymore, better to just use a span + css